package.json

```diff
@@ -102,7 +103,9 @@
     "eslint-plugin-jsx-a11y": "^6.3.1",
     "eslint-plugin-react": "^7.21.4",
     "html-webpack-plugin": "^4.5.0",
+    "husky": "^5.0.0-alpha.6",
     "jest": "^26.5.3",
+    "pinst": "^2.0.0",
```

Comment:
@webb04 given [this section](https://github.com/typicode/husky/tree/master#ci-servers) in the Husky documentation, do we need `pinst`?